src/pages/profiles/profiles.page.js

```diff
@@ -31,13 +32,15 @@ const ProfilesPage = () => {
       titleRef.current.scrollIntoView({ behavior: "smooth" });
   };
 
+  const { t } = useTranslation();
+
   if (isLoading) return <Spinner />;
 
   return (
     <main className="profiles">
       <Helmet>
         {/* <!-- Primary Meta Tags --> */}
-        <title>Публічні профілі – Immortal</title>
+        <title>{t("HOME.PUBLIC_PROFILES")} – Immortal</title>
         <meta name="title" content="Публічні профілі – Immortal" />
         <meta
           name="description"
```

Comment:
It looks as if you have not synchronized with the **develop** branch, please _rebase_ these changes, then _merge_ the current version from the **develop** branch, and then _push_ new changes

I think it will be like
```suggestion
      <DocumentHead
        title=`${t("HOME.PUBLIC_PROFILES")} – Immortal`
        description="Люди, про пам'ять яких, потурбувались їх родичі" // add new property to dictionary
        url="https://immortalv.com/profiles/"
      />
```